pool/app/guard/event.ml

```diff
@@ -21,10 +21,13 @@ let log_rules ~tags =
 type event =
   | DefaultRestored of Core.Authorizer.auth_rule list
   | RolesGranted of Repo.Uuid.Actor.t * Core.ActorRoleSet.t
+  | RoleRevoked of Repo.Uuid.Actor.t * Core.ActorRoleSet.t
```

Comment:
`Core.ActorRoleSet.t` can contain multiple roles? If so, please rename to `RolesRevoked` for consistency.